### DISCUSSION TITLE
note/basic/datatype

### DISCUSSION THREAD

**Discussion by giscus** [lkzwc/fe-ddu#94] [Announcements]
# note/basic/datatype

前端开发者流客的博客

https://lkzwc.netlify.app/note/basic/datatype.html

**Comment by lkzwc** [comment]
引用类型【对象】和【函数】

**Reply by lkzwc** [reply]
原始值  对象和函数

原始值内容不可变

**Reply by lkzwc** [reply]
let reaction = 'yikes';
reaction[0] = 'l';
console.log(reaction);  变量和值的场景

**Reply by lkzwc** [reply]
Nan不是数字 但是数字类型的值 typeof Nan > number
